main.go

```diff
@@ -28,24 +29,34 @@ func Main() int {
 	compact := flag.Bool("compact", false, "When printing code, use no indentation and most compact form")
 	showEval := flag.Bool("eval", true, "show eval results")
 	sharedState := flag.Bool("shared-state", false, "All files share same interpreter state (default is new state for each)")
+	configDir, err := os.UserConfigDir() // TODO: this is ugly on a mac... allow homedir...
+	historyFileDefault := filepath.Join(configDir, ".grol_history")
```

Comment:
```suggestion
	historyFileDefault := filepath.Join(configDir, "grol", "history")
```

Reply:
and now I have to mkdir all etc...? isn't it _enough_ to use the config dir and put files there

Reply:
It's dirty. I would prefer if you added a .grol_history file at the root of home folder than creating a file at the root of config folder.

Reply:
ok so I switched back to homedir as default but added GROL_HISTORY_FILE env var for people who don't like it so they don't have to set it in the flag either, does that work?